components/fxa-client/src/ffi.rs

```diff
@@ -197,6 +207,17 @@ impl msg_types::Capabilities {
     }
 }
 
+unsafe fn get_buffer<'a>(data: *const u8, len: i32) -> &'a [u8] {
+    assert!(len >= 0, "Bad buffer len: {}", len);
+    if len == 0 {
+        // This will still fail, but as a bad protobuf format.
```

Comment:
This comment seems out of context now that the code has moved, since it's not longer adjacent to the stuff that does the protobuf decoding. May be worth just deleting it?